assets/css/style.scss

```diff
@@ -171,39 +164,18 @@
 			margin-right: auto;
 		}
 	}
-
 	@for $i from 2 to 9 {
 		&.has-#{$i}-columns .wc-block-grid__product {
 			flex: 1 0 calc(#{ 100% / $i });
 			max-width: 100% / $i;
 		}
 	}
-
-	.wc-block-grid__products {
-		font-size: 1em;
-
-		.wc-block-grid__product-title {
-			font-size: 1em;
-		}
-
-		.wc-block-grid__product-price {
```

Comment:
This rule was needed for Tweenty Nineteen too, otherwise text looks too big.

Reply:
It's bug compared to the shop page, but it doens't look broken so it's probably ok to inherit.